tests/workers/xmodel/test_resnet50.py

```diff
@@ -17,6 +17,7 @@
 import amdinfer
 import amdinfer.pre_post as pre_post
 import amdinfer.testing
+from amdinfer._amdinfer import *
```

Comment:
Don't import everything with *. You also shouldn't need to import `_amdinfer` either

Reply:
still need import InferenceRequest, InferenceRequestInput, DataType as we are constructing the request in the file